src/test/java/org/prebid/server/bidder/medianet/MedianetBidderTest.java

```diff
@@ -0,0 +1,142 @@
+package org.prebid.server.bidder.medianet;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import org.junit.Before;
+import org.junit.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+
+import java.util.List;
+import java.util.function.Function;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.banner;
+
+public class MedianetBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "https://test.media.net?src=external.prebidserver.com";
+    private static final BidRequest DUMMY_REQUEST = BidRequest.builder()
```

Comment:
We prefer to use givenBidRequest method, you can check examples in another test classes

Reply:
Changed to 'given' nomenclature